deploy/src/main/deploy/example-tenants.json

```diff
@@ -1,6 +1,10 @@
 [
   {
     "tenant-id": "DEFAULT_TENANT",
+    "trusted-ca": {
+      "subject-dn": "CN=ca,OU=Hono,O=Eclipse IoT,L=Ottawa,C=CA",
```

Comment:
could we also inject the CA's subject DN using the resources plugin?

Reply:
That would be more consistent.